libs/gi/db/src/Database/DataEntries/DisplayArtifactEntry.ts

```diff
@@ -46,6 +46,8 @@ export type FilterOption = {
   locked: Array<'locked' | 'unlocked'>
   rvLow: number
   rvHigh: number
+  mrvLow: number
```

Comment:
I think your implementation is flawed from the data perspective, you should have 3 data: `rvLow, rvHigh, useMaxRv`. Currently, the RV values are not persisted when you toggle between `useMaxRv`, and after page refresh, the value of the toggle is not persisted either.